### ISSUE TITLE
Enable compatibility with stylelint 16.x

### ISSUE THREAD

**Issue by swcm-mnestler** [bjankord/stylelint-config-sass-guidelines#299]
Like back in #273, stylelint 16 is released and this project's peer dependency constraints doesn't allow it. Several deprecated rules were removed in the new version, so it might take a bit of effort to upgrade.
```
npm ERR! While resolving: stylelint-config-sass-guidelines@10.0.0
npm ERR! Found: stylelint@16.0.1
npm ERR! node_modules/stylelint
npm ERR!   dev stylelint@"^16.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer stylelint@"^15.2.0" from stylelint-config-sass-guidelines@10.0.0
npm ERR! node_modules/stylelint-config-sass-guidelines
npm ERR!   dev stylelint-config-sass-guidelines@"10.0.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: stylelint@15.11.0
npm ERR! node_modules/stylelint
npm ERR!   peer stylelint@"^15.2.0" from stylelint-config-sass-guidelines@10.0.0
npm ERR!   node_modules/stylelint-config-sass-guidelines
npm ERR!     dev stylelint-config-sass-guidelines@"10.0.0" from the root project
```

**Comment by echoix** [comment]
The tool will need to be disabled in Megalinter to not hold back the other stylelint tools, like with the previous major release.

**Comment by bjankord** [comment]
stylelint v16.x support has been released in v11.0.0
